src/signTx.c

```diff
@@ -90,7 +90,7 @@ static inline void advanceStage()
 			        ctx->includeAuxData,
 			        ctx->includeValidityIntervalStart,
 			        ctx->includeMint,
-					ctx->includeScriptDataHash
+			        ctx->includeScriptDataHash
```

Comment:
The formatting error should be fixed in its respective branch (to avoid annoying merge conflicts). You are coding this alone, so you can force-push and rebase as needed.